src/main/scala/io/qbeast/spark/delta/IndexStatusBuilder.scala

```diff
@@ -55,7 +55,9 @@ private[delta] class IndexStatusBuilder(
 
     import spark.implicits._
     val ndims: Int = rev.transformations.size
-    revisionFiles
+    // TODO some files may not include metadata
+    val filesWithMetadata = revisionFiles.where("tags is not null")
```

Comment:
With the changes made, is this still necessary?

Reply:
I don't know if we should support the reading of files without Qbeast Metadata. This case would only happen if the user does compaction with Delta (or other underlying format). 
We can solve this in another PR.